Civi/Payment/PropertyBag.php

```diff
@@ -26,13 +26,23 @@ class PropertyBag implements \ArrayAccess {
 
   protected static $propMap = [
     'amount'                      => TRUE,
+    'total_amount'                => 'amount',
```

Comment:
This line may have caused a regression - https://lab.civicrm.org/dev/core/-/issues/3413 - it _appears_ to be treating `total_amount` as `amount` when both are passed - I think this was probably done by accident since I think `amount` was consolidated quite a long time ago